core/src/main/scala/org/apache/spark/deploy/rest/RestSubmissionClient.scala

```diff
@@ -411,7 +411,7 @@ private[spark] object RestSubmissionClient {
 
   // SPARK_HOME and SPARK_CONF_DIR are filtered out because they are usually wrong
   // on the remote machine (SPARK-12345) (SPARK-25934)
-  private val BLACKLISTED_SPARK_ENV_VARS = Set("SPARK_ENV_LOADED", "SPARK_HOME", "SPARK_CONF_DIR")
+  private val EXCLUDED_SPARK_ENV_VARS = Set("SPARK_ENV_LOADED", "SPARK_HOME", "SPARK_CONF_DIR")
```

Comment:
I see 'excluded', 'reject', 'inaccessible', etc used as new terms; the idea is to use terms more specific to the usage? OK if so. Like sometimes it's a list of things that are to be excluded, things that are inaccessible, etc.

Reply:
yes, the idea was to change it to be something specific for how it was used.  If you have other ideas please let us know.